home_page.php

```diff
@@ -71,26 +71,27 @@
 				<!-- Song slider container -->
 				<div class = "fit_slider">
 					<div class="slider_main">
+						
 						<!-- Slider for main song images/content -->
 						<div class="slider slider-for">
 						</div>
 						
-						<!-- Slider navigation thumbnails (no images currently in placeholders) -->
+						<!-- Slider navigation thumbnails -->
 						<div class="slider slider-nav">
 							<div>
-								<h3><img></h3>
+								<h3><img src = "Images/neon-street-light.jpg"></h3>
```

Comment:
I added navigation thumbnail images for the song slider to improve visual navigation.